### PR TITLE
Add indent and unindent

### PR DESCRIPTION
**Pull request by florianduros**

### CODE REVIEW

File: platforms/web/lib/useListeners/utils.ts [hunk 1]
```diff
@@ -42,5 +42,6 @@ export function mapToAllActionStates(
                 key.substring(1)) as ActionTypes
         ] = value.toLowerCase() as ActionState;
     }
+    console.log('actionStatesMap', actionStatesMap);
```
Comment (alunturner) [inline platforms/web/lib/useListeners/utils.ts:45]: This is probably just leftover from integration work

File: platforms/web/lib/constants.ts [hunk 2]
```diff
@@ -28,4 +28,6 @@ export const ACTION_TYPES = [
     'link',
     'codeBlock',
     'quote',
+    'indent',
+    'unIndent',
```
Comment (alunturner) [inline platforms/web/lib/constants.ts:32]: Does this have to be `unIndent` for any reason? To match something from the rust model? I find it jarring, but if it has to be that way, it has to be that way

Comment (florianduros) [reply platforms/web/lib/constants.ts:32]: Yes, it's needed to match the state returned by the rust model

File: platforms/web/lib/useListeners/utils.ts (hunk 1 above)
Comment (florianduros) [reply platforms/web/lib/useListeners/utils.ts:45]: Good catch !

**Review by alunturner — APPROVED** [review]
Not quite sure why SonarCloud thinks all the test cases are repetition in that file

**Comment by florianduros** [comment]
Also, it seems only work in the case of list. You can't indent outside of list

**Pull request merged**